core/src/main/java/org/polypheny/db/sql/ddl/altertable/SqlAlterSourceTableAddColumn.java

```diff
@@ -104,8 +104,8 @@ public void unparse( SqlWriter writer, int leftPrec, int rightPrec ) {
     public void execute( Context context, Statement statement ) {
         CatalogTable catalogTable = getCatalogTable( context, table );
 
-        if ( catalogTable.isView() ) {
-            throw new RuntimeException( "Not possible to use ALTER TABLE with Views" );
+        if ( catalogTable.isView() || catalogTable.isMaterialized() ) {
```

Comment:
isView() should be true for both, views and materialized views

Reply:
Changed and now it is checked if it is a table and otherwise it throws an error message.